rules-emerging-threats/2023/TA/Cozy-Bear/win_security_apt_cozy_bear_scheduled_tasks_name.yml

```diff
@@ -0,0 +1,47 @@
+title: Scheduled Tasks Names Used By SVR For GraphicalProton Backdoor
+id: 8fa65166-f463-4fd2-ad4f-1436133c52e1
+related:
+    - id: 2bfc1373-0220-4fbd-8b10-33ddafd2a142
+      type: similar
+status: experimental
+description: Hunts for known SVR-specific scheduled task names
+author: CISA
+references:
+    - https://www.cisa.gov/news-events/cybersecurity-advisories/aa23-347a
+date: 2023/12/18
+tags:
+    - attack.persistence
+logsource:
+    service: security
+    product: windows
+detection:
+    selection:
+        EventID:
+            - 4698
+            - 4699
+            - 4702
+        TaskName:
+            - '\defender'
```

Comment:
Is this specific enough (e.g. case insensitive matching)?

Reply:
It was taken directly from the report. They had both `Defender` and `defender`. And since its a ET rule, i didn't care a lot and by default there is not task named on windows like this. So i kept it